azuredevops.go

```diff
@@ -1,24 +1,24 @@
 package main
 
 import (
-	"fmt"
-	"os"
-	"strconv"
-
 	"crypto/tls"
-	"encoding/json"
-	"io/ioutil"
+	"fmt"
 	"net/http"
 	"net/url"
+	"os"
 	"strings"
 
 	"github.com/gin-gonic/gin"
 	"github.com/iver-wharf/wharf-api-client-go/pkg/wharfapi"
+	"github.com/iver-wharf/wharf-core/pkg/ginutil"
+	"github.com/iver-wharf/wharf-core/pkg/problem"
 	_ "github.com/iver-wharf/wharf-provider-azuredevops/docs"
+	"github.com/iver-wharf/wharf-provider-azuredevops/pkg/requests"
 )
 
 const apiRepositories string = "_apis/git/repositories"
 const apiProjects string = "_apis/projects"
+const apiProviderName string = "azuredevops"
 const itemsPath string = "items"
 const refsPath string = "refs"
```

Comment:
nit: These are becomming so many that I suggest grouping them into a single `const` statement for extra readability as that aligns the values on `gofmt`, and perhaps dropping the `string` type as that's inferred

```suggestion
const (
	apiRepositories = "_apis/git/repositories"
	apiProjects     = "_apis/projects"
	apiProviderName = "azuredevops"
	itemsPath       = "items"
	refsPath        = "refs"
)
```